robottelo/decorators.py

```diff
@@ -321,6 +322,12 @@ def __call__(self, func):
         :param func: The function being decorated.
 
         """
+        if self.bug_type not in ('bugzilla', 'redmine'):
```

Comment:
This will fail when importing the test module. Will avoid running the test until the expected bug_type is passed. Previously it was being checked just when running the test. In this case is better to fail earlier.